pyproject.toml

```diff
@@ -33,7 +33,6 @@ python = ">=3.7"
 asyncio-dgram = "1.2.0"
 click = ">=7.1.2,<9"
 dnspython = "2.1.0"
-typing-extensions = "^4.0.1"
```

Comment:
Move this into the dev dependencies, don't just remove it completely. The only reason this isn't failing now is the fact that `typing-extensions` is still getting installed because it's required by other libraries which we require, but we should still be explicit about this and put it into the dev dependencies section instead of just removing it.